sdk/core/azure-core/src/main/java/com/azure/core/http/rest/Substitution.java

```diff
@@ -53,4 +55,14 @@ public int getMethodParameterIndex() {
     public boolean shouldEncode() {
         return shouldEncode;
     }
+
+    /**
+     * Get whether or not the value from the method argument needs to be sent as string in case the
+     * values are a list or as multiple query parameters.
+     * @return Whether or not this query parameter list values should be sent as individual query
+     * params or as a single Json.
+     */
+    public boolean shouldDoMultipleQueryParams() {
```

Comment:
Not a fan of the name of this API. Thoughts on inverting it, `shouldMergeQueryParams`. I'm also not certain if this should be added to this class given substitution is generic for all param types (header, query, path, body, etc) and this value is specific to query parameters only.